pluribus/games/short_deck/state.py

```diff
@@ -219,7 +219,7 @@ def info_set(self) -> str:
         eval_cards = tuple([card.eval_card for card in cards])
         cards_cluster = self.info_set_lut[self._betting_stage][eval_cards]
         action_history = [str(action) for action in self._history]
-        return f"cards_cluster={cards_cluster}, history={action_history}"
+        return f"betting_stage={self._betting_stage}, cards_cluster={cards_cluster}, history={action_history}"
```

Comment:
How do we have the betting stage? The card cluster is not unique for each round in case that's what you are thinking (I can update that if needed). I was trying to think if the action sequence was alone unique. I think it might be (in which case we are good) but haven't had time to think it though. In any event, this above was just for me to be able to read/debug. I am unsure if it is of any benefit to the bot to be honest.